app/presenters/specialist_document_presenter.rb

```diff
@@ -26,6 +26,10 @@ def show_metadata_block?
     content_item.finder.show_metadata_block
   end
 
+  def hide_contents_list?
+    content_item.finder.hide_contents_list
+  end
```

Comment:
I've mentioned this in the other PRs, but this method should be renamed to `show_contents_list?` with the appropriate change in logic.